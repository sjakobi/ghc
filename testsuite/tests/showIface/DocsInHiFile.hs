{-| `elem`, 'print',
`Unknown',
'<>', ':=:', 'Bool'
-}
module DocsInHiFile where

elem :: ()
elem = ()
