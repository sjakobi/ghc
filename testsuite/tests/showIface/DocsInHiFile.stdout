doc names map:
  ":=:":
  "<>":
    <> from GHC.Base
  "Bool":
    Bool from GHC.Types
  "Unknown":
  "elem":
    elem from Data.Foldable
    elem from DocsInHiFile
  "print":
    print from System.IO
module header:
  Just text:
         " `elem`, 'print',
`Unknown',
'<>', ':=:', 'Bool'
"
       spans: 2-6, 10-15, 19-26, 30-32, 36-39, 43-47
